### PR TITLE
Create 2023 - June 6th.md

### PR DESCRIPTION
**Pull request by vincent6767**

### CODE REVIEW

File: meetings/notes/2023 - June 6th.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+2. On the Join TSC meeting on May 30th, 2023, Royal proposes to have the retrospective session together with TSC and other SIG members during one of the office hours. The agenda for today is to decide the following action steps, whether to hold the session with other members or do it ourselves.
+   1. SIG Release agrees with this.
+   2. @tonybalandiuk to pick one of the TSC meeting hours and then gets approval from TSC to run the retrospective session on the meeting.
+3. Decides when we are going to decide the next release date.
+   1. The next release will be harder as Amazon members won't participate much in the next release. The next big question is the QA resources.
+   2. @joe will talk with Nicole about this and report back the result.
+4. SIG Release Election
+   1. @vincent to figure out the election process and report back in the SIG release channel.
+
+## Action Items
```
Comment (tonybalandiuk) [inline meetings/notes/2023 - June 6th.md:26]: 1 more action item: 
tonybalandiuk to pick one of the TSC meeting hours and then gets approval from TSC to run the retrospective session on the meeting.

**Review by tonybalandiuk — APPROVED** [review]
Approving. Looks good. The summary of action items at the bottom is missing one of the action items, but it's captured above in the notes. Overall this is ok and we can submit.

File: meetings/notes/2023 - June 6th.md (hunk 1 above)
Comment (vincent6767) [reply meetings/notes/2023 - June 6th.md:26]: Thanks for catching the missing action items! I'll make updates today

Comment (vincent6767) [reply meetings/notes/2023 - June 6th.md:26]: Done.

**Pull request merged**